src/commands/codepush/patch.ts

```diff
@@ -0,0 +1,101 @@
+import { AppCommand, CommandArgs, CommandResult, help, failure, ErrorCodes, success, getCurrentApp, shortName, longName, required, hasArg, position, name } from "../../util/commandline";
+import { out } from "../../util/interaction";
+import { inspect } from "util";
+import { MobileCenterClient, models, clientRequest, clientCall } from "../../util/apis";
+import * as chalk from "chalk";
+import * as semver from "semver";
+
+const debug = require("debug")("mobile-center-cli:commands:codepush:patch");
+
+@help("Update the metadata for an existing release")
+export default class PatchCommand extends AppCommand {
+  
+  @help("CodePush deployment name.")
+  @required
+  @name("ExistingDeploymentName")
+  @position(0)
+  public deploymentName: string;
+
+  @help("Label of the release to update. Defaults to the latest release within the specified deployment.")
+  @required
+  @name("ExistingReleaseLabel")
+  @position(1)
+  public releaseLabel: string;
+
+  @help("Specifies whether this release should be considered mandatory. Putting -m flag means mandatory.")
+  @shortName("m")
+  public isMandatory: boolean;
+
+  @help("Specifies whether this release should be immediately downloadable. Putting -x flag means disabled.")
+  @shortName("x")
+  public isDisabled: boolean;
+
+  @help("Semver expression that specifies the binary app version(s) this release is targeting (e.g. 1.1.0, ~1.2.3).")
+  @shortName("t")
+  @longName("targetBinaryVersion")
```

Comment:
I'd prefer if you went to the kebab-case version, but if not at least be consistent on using either initial capital letters or lowercase.

Reply:
fixed.